src/super_gradients/common/object_names.py

```diff
@@ -282,3 +282,9 @@ class ConcatenatedTensorFormats:
     LABEL_NORMALIZED_XYXY = "LABEL_NORMALIZED_XYXY"
     LABEL_NORMALIZED_XYWH = "LABEL_NORMALIZED_XYWH"
     LABEL_NORMALIZED_CXCYWH = "LABEL_NORMALIZED_CXCYWH"
+
+
+class CollateFunctions:
+    DEFAULT_COLLATE_FN = "DEFAULT_COLLATE_FN"
```

Comment:
You missed some collate fns (mixup, mutiscale...cmd+shift+f collate and add the missing ones please).